src/actors/lead.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from thespian.actors import ActorAddress
+
+from src.actors.generic import GenericActor
+from src.utils.constants import ActorNames, Requests
+
+
+class LeadActor(GenericActor):
```

Comment:
Is this meant to be the main actor? If so a more specifc name would be better, i.e. "BellboyActor"

Reply:
Lead actor is fitting, and it's how I've been describing it in our convos.

Reply:
Generic Actor --> Bellboy Actor is good too, then Lead Actor --> BellboyLeadActor